rfd/0017-supply-chain-hardening.md

```diff
@@ -0,0 +1,59 @@
+# Supply Chain Hardening
```

Comment:
I'll write an RFD for RFD format.
For now, let's keep it consistent to existing RFDs.

```suggestion
---
authors: Alexander Klizhentas (sasha@goteleport.com)
state: draft
---


# RFD 17 - Supply Chain Hardening
```